index.html

```diff
@@ -323,7 +324,7 @@ <h1 class="fisto">Contact me</h1><br>
   <div class="popup-container">
     <div class="popup">
       <div class="popup-header">
-        <h2 id="popup-title"> </h2>
+        <h2 id="popup-title"></h2>
         <i id="close-popup" class="fas fa-times"></i>
       </div>
       <div class="popup-details">
```

Comment:
- Kindly follow the design guidelines and use the required colors for hover effect in your submit button.

**Yours:**
<img width="1440" alt="Screen Shot 2021-11-11 at 5 05 56" src="https://user-images.githubusercontent.com/22258274/141287765-a158022f-7598-4e7d-b86c-daaeba63575d.png">


**Template:**
<img width="1440" alt="Screen Shot 2021-11-11 at 5 03 12" src="https://user-images.githubusercontent.com/22258274/141287844-3d0bab32-9af5-4897-a3e6-4fa1b2a02181.png">